src/labels_by_user_input.py

```diff
@@ -10,25 +12,60 @@
 )
 
 
+LABELS_SET = {
```

Comment:
@rnetser @AdiZav 
what do you think about not uning /un for negative?
something like /unlgtm looks wired

maybe /-lgtm
/-verified

or any other suggestions

Reply:
Yeah it looks better imo.
Do you mind if we do it in a seperate PR?
This pr has lots of conflicts whenever something's merged to labels_by_user_input

Reply:
you shouldn't have any conflict, this is your code :)

Reply:
I mean changes have been made to this file in the master branch, and since this pr refactors a big chunk of this file, every change raises conflicts

Reply:
But you are rebased and the latest code right? 
if so then you shouldn't have any conflicts since you are the only one that change it right now.

Reply:
Sure ig.
So we're going with /-?
Another thing I had in mind was !verified but I guess it just adds unnecessary hassle

Reply:
@rnetser what do you think?

Reply:
I agree - `/-` is better

Reply:
great, @AdiZav let make it /-

Reply:
ACK

Reply:
ACK